README.md

```diff
@@ -1,72 +1,92 @@
+- Telegram
+	- [Announcement Channel](https://t.me/DarwiniaNetwork_Announcement)
+	- [Official Group](https://t.me/DarwiniaNetwork)
+	- [Official Group (CN)](https://t.me/DarwiniaNetworkCN)
+	- [Official Technical Group](https://t.me/DarwiniaDev)
+	- [Unofficial Group](https://t.me/darwinianetworkpricespeculation)
+- Matrix
+	- [Official Technical Room](https://matrix.to/#/#darwinia:matrix.org)
+- Discord
+	- [Official Server](https://discord.gg/uqa3snSGTj)
+- Slack
+	> Technical members are more active on this place. If you expect a quick response, here you go.
+	- PM [Support](mailto:support@darwinia.network) or [Xavier](mailto:xavier.lau@itering.com)
+- Reddit
+	- [Community](https://www.reddit.com/r/DarwiniaCommunity)
+- Twitter
+	- [Official Account](https://twitter.com/DarwiniaNetwork)
+- Medium
+	- [Official Account](https://darwinianetwork.medium.com)
+- Wechat
+	- Official Group: Add "**DarwiniaBot**" to get the link
+	- Official Account: "**DARWINIA**"
+- Weibo
+	- [Official Account](https://weibo.com/DarwiniaNetwork)
+
+## Installation
+- Downloading pre-built binary from **[releases](https://github.com/darwinia-network/darwinia/releases)** page.
+- Using the docker image on **[releases](https://github.com/darwinia-network/darwinia/releases)** page.
+- Building from source follow this **[guide](#build-from-source)**.
 
 ## Building from Source
+> Make sure that you have the dependencies. Follow [substrate-getting-started](https://substrate.dev/docs/en/knowledgebase/getting-started).
 
-> NOTE: Substrate development for windows is not properly supported! It is highly recommend to use Windows Subsystem Linux (WSL) and follow the instructions for Ubuntu/Debian. Check out [Getting Started on Windows](https://substrate.dev/docs/en/knowledgebase/getting-started/windows-users) for more detail.
-
-For Unix-based user, just grab the source code and build it. Make sure you have Rust and the support software installed.
-
-Install Rust first:
-
+### Installing via Cargo
 ```sh
-curl https://sh.rustup.rs -sSf | sh
+cargo install --git https://github.com/darwinia-network/darwinia --tag <version> --locked
 ```
 
+### Building via Source
 ```sh
-rustup default nightly
-rustup target add wasm32-unknown-unknown
+gh repo clone darwinia-network/darwinia
+git checkout <version>
+cargo build --release
 ```
 
-Additional packages may also be required:
-
-- Linux:
-
-	```sh
-	sudo apt install cmake pkg-config libssl-dev git clang libclang-dev
-	```
-
-- Mac:
-
-	```sh
-	brew install cmake pkg-config openssl git llvm
-	```
-
-Then, clone the Darwinia source code and build the target:
+## Networks
+This repository supports runtimes for Darwinia and Crab.
 
+### Connecting to Darwinia Mainnet
+Connecting to the global Darwinia network by running:
 ```sh
-# Fetch the code
-git clone https://github.com/darwinia-network/darwinia.git
-cd darwinia
-
-# Build the node (The first build will be long (~30min))
-cargo build --release
+./darwinia --chain darwinia
```

Comment:
Recommend to give a more specific command or just a link to `How to run node` Chapter of the Darwinia/Crab doc.
It would be a little difficult for a beginner to master sub-command usage. So, The best solution is to give them a copy-paste command.